config/php-formatter.php

```diff
@@ -89,6 +89,9 @@
         'trim_array_spaces' => true,
         'whitespace_after_comma_in_array' => true,
     ],
+    // You can initialises your own ParallelConfig class here with your desired parameters
+    // default is set to detect the environment and use the appropriate resources
+    // 'parallel' => PhpCsFixer\Runner\Parallel\ParallelConfigFactory::detect(),
```

Comment:
should this line be commented?

Reply:
yep, by default we can just leave it 'undefined' which will end up using the default detection https://github.com/ComLaude/php-formatter/pull/4/files#diff-a7f7f4b6ef0ee4ae3d05932f2a7109985e234ceee3b84c24060b487dcb403983R46

I just wanted to make sure it's clear in the readme how you use this